interface.js

```diff
@@ -0,0 +1,204 @@
+$(document).ready(function() {
+  var game = new Game();
+  updateScore();
+  var frame1_roll1;
+  var frame1_roll2;
+  var frame2_roll1;
+  var frame2_roll2;
+  var frame3_roll1;
+  var frame3_roll2;
+  var frame4_roll1;
+  var frame4_roll2;
+  var frame5_roll1;
+  var frame5_roll2;
+  var frame6_roll1;
+  var frame6_roll2;
+  var frame7_roll1;
+  var frame7_roll2;
+  var frame8_roll1;
+  var frame8_roll2;
+  var frame9_roll1;
+  var frame9_roll2;
+  var frame10_roll1;
+  var frame10_roll2;
+
+
+  $('#frame1_roll1').change(function() {
+    roll1_input = $('#frame1_roll1').val();
+    frame1_roll1 = Number(roll1_input);
+  })
+
+  $('#frame1_roll2').change(function() {
+    roll2_input = $('#frame1_roll2').val();
+    frame1_roll2 = Number(roll2_input);
+  })
+
+  $('#frame1').click(function() {
+    console.log(frame1_roll1);
+    console.log(frame1_roll2);
+    game.frame_score(frame1_roll1, frame1_roll2);
+    updateScore();
+    return false;
+  })
+
+  $('#frame2_roll1').change(function() {
+    roll1_input = $('#frame2_roll1').val();
+    frame2_roll1 = Number(roll1_input);
+  })
+
+  $('#frame2_roll2').change(function() {
+    roll2_input = $('#frame2_roll2').val();
+    frame2_roll2 = Number(roll2_input);
+  })
+
+  $('#frame2').click(function() {
+    game.frame_score(frame2_roll1, frame2_roll2);
```

Comment:
Could may add a condition in here which stops the frame's score exceeding 10. Butttttt, I do think there would be better ways to do this!

Reply:
agree! Was thinking of just restricting it in the front end